WordPress/src/main/java/org/wordpress/android/ui/prefs/DotComSiteSettings.java

```diff
@@ -277,6 +316,44 @@ public void onErrorResponse(VolleyError error) {
                 });
     }
 
+    private void fetchJetpackModuleSettings() {
+        ++mFetchRequestCount;
+        WordPress.getRestClientUtilsV1_1().getJetpackModuleSettings(
+                mSite.getSiteId(), new RestRequest.Listener() {
+                    @Override
+                    public void onResponse(JSONObject response) {
+                        AppLog.v(AppLog.T.API, "Received Jetpack module settings");
+                        if (response == null) return;
+                        JSONArray array = response.optJSONArray("modules");
+                        if (array == null) return;
```

Comment:
Not sure, should we try to emit `onFetchResponseReceived` before those `return` statements?

Reply:
makes sense! I checked the other calls and they seem to do that. I'll change the logic to always call the `onFetchResponseReceived`